services/actions/src/rpc/genSchemasDocs.js

```diff
@@ -53,53 +54,88 @@ const setMarkdownDocMutation = `
   }
 `;
 
+const getMemberFieldsMapper = source => param => {
+  const entry = Object.entries(param)[0];
+
+  if (!entry) {
+    return null;
+  }
+
+  const [key, title] = entry;
+
+  if (!source[key]) {
+    return null;
+  }
+
+  if (Array.isArray(source[key])) {
+    return `${text(title, { bold: true, indent: 2, postNewLine: false })}: \`${source[key].join(', ')}\``;
+  }
+
+  return `${text(title, { bold: true, indent: 2, postNewLine: false })}: \`${source[key]}\``;
+};
+
+const generateMemberDoc = member => {
+  const { meta } = member;
+
+  let result = header(member.shortTitle, { size: 5, bold: true, indent: 8 });
+  result += text(member.description || 'No description provided', { indent: 8 });
```

Comment:
there could be `description` for the cube itself too, and we have to support this